### DISCUSSION TITLE
Δημιουργία ομάδων

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/hci#1952] [Announcements]
### Για να φτιάξετε την ομάδα σας (Προθεσμία τέλος 1ης εβδομάδα, δλδ Κυριακή 8 Οκτωβρίου)

1. Η [δημιουργία και ο συντονισμός ομάδας](https://epidrome.github.io/teaching/team/) είναι μια δύσκολη τέχνη, για αυτό, εκτός από τις οδηγίες, σας δίνουμε αρκετό χρόνο και σχετικό πλαίσιο (=παράλληλη αποστολή εισαγωγής στις συζητήσεις της βοήθειας, ώστε να γίνει η γνωριμία).
2. Για να δημιουργήσετε μια νέα ομάδα, αφού πρώτα φτιάξετε έναν [νέο λογαριασμό οργανισμού](https://docs.github.com/en/organizations), κάντε ένα νέο σχόλιο αντιγράφοντας τον παρακάτω πίνακα και περιμένετε να απαντήσουν στο σχόλιο σας.
3. Μπορείτε να εντοπίσετε και να καλέσετε νέα μέλη από [την εισαγωγή της 1ης εβδομάδας](https://courses-ionio.github.io/help/intro/) [που θα βρείτε στις συζητήσεις](https://github.com/courses-ionio/hci/discussions/categories/show-and-tell).
4. Αν σας αρέσει κάποια ομάδα που υπάρχει ήδη στη συζήτηση παρακάτω, ρωτήστε αν μπορείτε να μπείτε και αν καμία ομάδα δεν είναι διαθέσιμη, τότε φτιάξτε μια νέα ομάδα.
5. Μην ξεχνάτε να ενημερώνετε τα στοιχεία των ομάδων σε περίπτωση αλλαγών και την κατάσταση της ομάδας, έτσι ώστε να φαίνεται πάντα ποιες είναι ενεργές και ποια είναι τα μέλη.
6. Όπως γράφουν και οι αντίστοιχες οδηγίες θα εκτιμηθεί θετικά η δημιουργία ανοικτών ομάδων με διαφανείς κανόνες (π.χ., είσοδος-έξοδος μέλους) και διαδικασίες (που θα ορίσετε εσείς) αντί της δημιουργίας κλειστών ομάδων με προφορικές και κλειστές διαδικασίες, γιατί αυτός είναι ο σύγχρονος (και μελλοντικός) τρόπος (συν)εργασίας.
7. Μια ομάδα θα πρέπει να έχει τουλάχιστον πέντε μέλη, άρα αν το μάθημα επιλογής έχει λιγότερα από δέκα άτομα τότε όλη η τάξη θα είναι μια ομάδα

Η δημιουργία και ο συντονισμός της ομάδας είναι δική σας ευθύνη και θα γίνει σε νέο προφίλ οργανισμού που θα δημιουργήσετε στο γκιτχαμπ και θα γράψετε παρακάτω:

Μετά την δημιουργία της ομάδας αντιγράψτε και συμπληρώστε τον παρακάτω πίνακα στην πρώτη ανάρτηση της ομάδας σας παρακάτω:

```
Όνομα ομάδας: delta
GitHub organisation url: [delta](url)

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |
```

Παρακαλούμε χρησιμοποιήστε την παρακάτω περιοχή αποκλειστικά για την οργάνωση ομάδων και όχι για απορίες.

**Comment by GeorgeChat** [comment]
Όνομα ομάδας: Eternals Kit
Organisation URL: [Eternals Kit](https://github.com/EternalsKit)
Νέα Μέλη Δεκτά: ΝΑΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | Π2012070 | [GeorgeChat](https://github.com/GeorgeChat) |
| 2 | Π2015191 | [difrag](https://github.com/difrag) |
| 3 | Π2019024 | [P19GKOU](https://github.com/P19GKOU) |
| 4 | inf2022181 | [inf2022181](https://github.com/inf2022181) |

**Reply by difrag** [reply]
γεια σ γιωργο, νομιζω χρειαζεσαι ατομα
κ εγω :P

**Reply by GeorgeChat** [reply]
@difrag Σε ευχαριστώ για το ενδιαφέρον σου, σου έχω στείλει invite. Όταν κάνεις accept, παρακαλώ άσε ένα σχόλιο στα Discussions του οργανισμού.

**Reply by P19GKOU** [reply]
@GeorgeChat Ενδιαφέρομαι επίσης για την ομάδα

**Reply by GeorgeChat** [reply]
@P19GKOU Ευχαριστώ για το ενδιαφέρον, έχεις invite. Παρακαλώ μπες στα Discussions του οργανισμού και άσε κι εσύ ένα σχόλιο.

**Reply by inf2022181** [reply]
Μπορώ να μπώ και εγώ στην ομάδα;

**Reply by GeorgeChat** [reply]
@inf2022181 Βεβαίως. Παρακαλώ μπες κι εσύ στα Discussions και σχολίασε κάτω από το θέμα Επικοινωνία.

**Comment by ChrysaDima19** [comment]
Όνομα ομάδας: Epikoinonites
GitHub organisation url: https://github.com/Epikoinonites
Νέα μέλη δεκτά: NAI

| --- | --- | --- |
| # |     ΑΜ     |  GitHub profile | 
| 1 | 2022044 | [ChrysaDima19](https://github.com/ChrysaDima19) |
| 2 | 2022083 | [PafsaniasKontos](https://github.com/PafsaniasKontos) | 
| 3 | 2022182 | [ChristoSarantidis](https://github.com/ChristoSarantidis) | 
| 4 | 2022202 | [AgnesTatsis](https://github.com/AgnesTatsis) |
| 5 | 2022076 | [chriskarydis](https://github.com/chriskarydis) |
| 6 | 2022118 | [vmillios](https://github.com/vmillios) |
| 7 | 2022105 | [eisvo12](https://github.com/eisvo12) |

**Reply by eisvo12** [reply]
Να μπω και εγω στην ομαδα?

**Reply by ghost** [reply]
Να μπω και εγω στην ομαδα?

**Reply by ChrysaDima19** [reply]
Ναι φυσικά!

**Reply by GeorgeChat** [reply]
Καλημέρα, 
έπιτα από επικοινωνία της ομάδας μας με τον κο. Χωριανόπουλο, έγινε πρόταση από τον ίδιο για συγχώνευση των οργανισμών μας. Όποτε σας είναι εφικτό και επιθυμητό, παρακαλώ στείλτε πρόσκληση προς τον οργανισμό σας στα μέλη μας:

| # | Github profile |
| -- | -- |
| 1 | [GeorgeChat](https://github.com/GeorgeChat) |
| 2 | [P19GKOU](https://github.com/P19GKOU) |
| 3 | [inf2022181](https://github.com/inf2022181) |

Ευχαριστούμε εκ των προτέρων

**Reply by chriskarydis** [reply]
Δημιουργήθηκε καινούργιο comment section με την ομάδα και τα καινούργια μέλη. Αυτό το comment μπορεί να διαγραφεί καθώς δεν είναι έγκυρο και έχει δημιουργηθεί από χρήστη που δεν είναι πια μέλος του οργανισμού.

**Comment by chriskarydis** [comment]
Όνομα ομάδας: Epikoinonites
GitHub organisation url: https://github.com/Epikoinonites
Νέα μέλη δεκτά: NAI



| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2022076 | [chriskarydis](https://github.com/chriskarydis) |
| 2 | 2022202| [AgnesTatsis](https://github.com/AgnesTatsis) |
| 3 | 2022182  | [ChristoSarantidis](https://github.com/ChristoSarantidis) |
| 4 | 2022083 | [PafsaniasKontos](https://github.com/PafsaniasKontos) |
| 5 | p12chat | [GeorgeChat](https://github.com/GeorgeChat) |
| 6 | p19gkou| [P19GKOU](https://github.com/P19GKOU) |
| 7 | inf2022181| [inf2022181](https://github.com/inf2022181) |
